node/src/BaseClient.ts

```diff
@@ -1092,6 +1095,77 @@ export class BaseClient {
         return this.createWritePromise(createDel(keys));
     }
 
+    /**
+     * Serialize the value stored at `key` in a Valkey-specific format and return it to the user.
+     *
+     * @See {@link https://valkey.io/commands/dump/|valkey.io} for details.
+     *
+     * @param key - The key of the set.
```

Comment:
"of the set" isn't right? 

Can you also fix Java documentation too: https://github.com/valkey-io/valkey-glide/blob/main/java/client/src/main/java/glide/api/commands/GenericBaseCommands.java#L1156

Reply:
please use Python to fix both: https://github.com/valkey-io/valkey-glide/blob/main/python/python/glide/async_commands/core.py#L5982